py-polars/tests/unit/dataframe/test_df.py

```diff
@@ -3629,6 +3595,67 @@ def test_ufunc_multiple_expressions() -> None:
     assert_series_equal(expected, result)  # type: ignore[arg-type]
 
 
+def test_unstack() -> None:
+    from string import ascii_uppercase
+
+    df = pl.DataFrame(
+        {
+            "col1": list(ascii_uppercase[0:9]),
+            "col2": pl.int_range(0, 9, eager=True),
+            "col3": pl.int_range(-9, 0, eager=True),
+        }
+    )
+    assert_frame_equal(
+        df.unstack(step=3, how="vertical"),
+        pl.from_repr(  # type: ignore[arg-type]
```

Comment:
Can't we just define the dataframes normally? I'm not a big fan of littering our tests with these big string representations.

Reply:
Lol, sure, not a problem; sometimes I like the clarity of seeing exactly what results I'm expecting, but it is admittedly overkill most of the time 😅